Project.toml

```diff
@@ -12,6 +12,7 @@ Gtk = "4c0ca9eb-093a-5379-98c5-f87ac0bbbf44"
 GtkReactive = "27996c0f-39cd-5cc1-a27a-05f136f946b6"
 Hungarian = "e91730f6-4275-51fb-a7a0-7064cfbd3b39"
 Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
+Interact = "c601a237-2ae4-5e1e-952c-7a85b0c7eef1"
```

Comment:
Please make sure to add a line to set bounds for Interact.

Reply:
For clarity, the bounds would be set in the Project.toml. Here is more info on bounds: https://julialang.github.io/Pkg.jl/v1/compatibility/

Reply:
Okay so I am not well familiar with the compat stuff(mostly had the bot handle it). Should I set it to the current version of Interact.jl?

Reply:
What I normally try to do is having a look at the changelog of the package or the documentation to figure out the first version that might support this and then test it. Or if the current version is relatively old (like 3 month) I suppose that it's good enough to just support their latest version 😁
We don't support Julia v1.0 so I figure people who use our package are normally relatively up to date when possible.

Reply:
I think the bound you picked for Interact is completely reasonable given that it was last tagged for a release back in 2019. Thanks Sov.

Reply:
Sounds cool!